README.md

```diff
@@ -141,11 +155,11 @@ We use [SemVer](http://semver.org/) for versioning. For the versions available,
 
 ## License
 
-*Required - Identify the project's license here.*
+*Required - Create a file that contains at least the licensing status of the code and the full text of the license or at least a link to where the license is officially maintained.*
```

Comment:
Include that the license file should be named LICENSE.md ; that way we don't have issues finding them if we need